### ISSUE TITLE
awesimsoss crashes when trying to do reshaping after simulate call with nresets = 1 in v.0.3.3

### ISSUE THREAD

**Issue by nespinoza** [spacetelescope/awesimsoss#80]
Hey @hover2pi,

 The following code crashes with the latest version of `awesimsoss`:

```
import numpy as np

import astropy.units as q

from awesimsoss import ModelTSO
import batman

rprs = 0.02037
wavelength, transit_depth = np.linspace(0.4,3.5,1000), (rprs**2)*np.ones(1000)

# First, define effective temperature, log(g), [Fe/H] and 
# 2MASS J-magnitude of the host star (the latter obtained from Simbad):
teff, logg, feh, jmag = 4599, 4.62, -0.06, 9.086

# Now define orbital properties of K2-141b, the exoplanet. First, orbital period and time-of-transit center:
P, mid_transit_time = 0.2803244, 57743.5716 + 2400000.5
# Scaled semi-major axis and planet radius (in units of stellar radius), inclination, eccentricity and 
# argument of periastron passage:
a, rp, inc, ecc, omega = 2.264, rprs, 86.3, 0., 90.

nintegrations, ngroups, start_time = 308, 3, '2022-12-6T23:14:44.87'

tsoframes = ModelTSO(ngrps=ngroups, nints=nintegrations, nresets = 1, teff = teff, logg = logg, 
                     feh = feh, jmag = jmag, stellar_model = 'ATLAS', obs_date = start_time, 
                     target = 'K2-141', run = False)

params = batman.TransitParams()
params.t0 = mid_transit_time # time of inferior conjunction
params.per = P # orbital period (days)
params.a = a # semi-major axis (in units of stellar radii)
params.rp = rp # radius ratio for Jupiter orbiting the Sun
params.inc = inc # orbital inclination (in degrees)
params.ecc = ecc # eccentricity
params.w = omega # longitude of periastron (in degrees) p
params.limb_dark = 'quadratic' # limb darkening profile to use
params.u = [0.1,0.1] # limb darkening coefficients; just initializations for now
supersample_factor = 10

tmodel = batman.TransitModel(params, tsoframes.time.jd, 
                             supersample_factor = supersample_factor, 
                             exp_time = tsoframes.frame_time*(1./(3600.*24.)))

# Stellar parameters --- will be useful to generate limb-darkening coefficients internally in 
# awesimsoss:
tmodel.teff = teff
tmodel.logg = logg
tmodel.feh = feh
# Define ATLAS9 as the model grid:
tsoframes.model_grid = 'ATLAS9'

transit_spectrum = [wavelength*q.um, transit_depth]
tsoframes.simulate(planet=transit_spectrum, tmodel=tmodel, params = params, supersample_factor = supersample_factor)
```

The traceback is:

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-9-670f8308ccef> in <module>
      6 if not os.path.exists(outfile):
      7     tsoframes.simulate(planet=transit_spectrum, tmodel=tmodel, 
----> 8                        params = params, supersample_factor = supersample_factor)

~/.local/lib/python3.7/site-packages/awesimsoss-0.3.3-py3.7.egg/awesimsoss/awesim.py in simulate(self, ld_coeffs, noise, n_jobs, params, supersample_factor, **kwargs)
    950         for order in self.orders:
    951             order_name = 'tso_order{}_ideal'.format(order)
--> 952             setattr(self, order_name, getattr(self, order_name).reshape(self.dims).astype(np.float64))
    953 
    954         # Make ramps and add noise to the observations

ValueError: cannot reshape array of size 645922816 into shape (308,3,256,2048)
```

I'm pretty sure this is a problem with me setting `nresets=1`, because when I set `nresets=0` there is no problem. This is important, I think, because the standard is to have `nresets=1`. I'm sure you'll probably have a quick solution to this, but I'll take a look at this anyways during the day.

Néstor

**Comment by hover2pi** [comment]
Should be fixed in #82

**Comment by nespinoza** [comment]
It is! I'll close this issue then. Thanks!
